server/db/user-migration.sql

```diff
@@ -0,0 +1,135 @@
+DROP TABLE IF EXISTS users;
+-- Create Users Table
+CREATE TABLE users (
```

Comment:
we should follow same naming scheme, i.e. not plural (programme, initiative, user)
```suggestion
CREATE TABLE user (
```